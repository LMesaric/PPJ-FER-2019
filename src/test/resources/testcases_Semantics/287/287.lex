KR_CONST 1 const
KR_INT 1 int
IDN 1 r
OP_PRIDRUZI 1 =
BROJ 1 33
TOCKAZAREZ 1 ;
KR_VOID 3 void
IDN 3 f
L_ZAGRADA 3 (
KR_VOID 3 void
D_ZAGRADA 3 )
L_VIT_ZAGRADA 3 {
KR_RETURN 4 return
TOCKAZAREZ 4 ;
D_VIT_ZAGRADA 5 }
KR_INT 6 int
IDN 6 main
L_ZAGRADA 6 (
KR_VOID 6 void
D_ZAGRADA 6 )
L_VIT_ZAGRADA 6 {
KR_INT 7 int
IDN 7 a
OP_PRIDRUZI 7 =
BROJ 7 666
TOCKAZAREZ 7 ;
KR_CONST 8 const
KR_CHAR 8 char
IDN 8 e
OP_PRIDRUZI 8 =
ZNAK 8 'e'
TOCKAZAREZ 8 ;
KR_CONST 9 const
KR_INT 9 int
OP_PRIDRUZI 9 =
IDN 9 e
TOCKAZAREZ 9 ;
KR_CHAR 11 char
IDN 11 t
TOCKAZAREZ 11 ;
KR_INT 12 int
IDN 12 s
OP_PRIDRUZI 12 =
BROJ 12 3
TOCKAZAREZ 12 ;
L_VIT_ZAGRADA 13 {
KR_INT 14 int
IDN 14 a
OP_PRIDRUZI 14 =
BROJ 14 5
TOCKAZAREZ 14 ;
KR_INT 15 int
IDN 15 y
OP_PRIDRUZI 15 =
IDN 15 e
PLUS 15 +
BROJ 15 1
TOCKAZAREZ 15 ;
KR_CONST 16 const
KR_INT 16 int
IDN 16 v
OP_PRIDRUZI 16 =
BROJ 16 666
TOCKAZAREZ 16 ;
IDN 20 s
OP_PRIDRUZI 20 =
IDN 20 y
PLUS 20 +
BROJ 20 1
TOCKAZAREZ 20 ;
D_VIT_ZAGRADA 21 }
IDN 22 s
OP_PRIDRUZI 22 =
IDN 22 a
PLUS 22 +
BROJ 22 1
TOCKAZAREZ 22 ;
KR_RETURN 23 return
BROJ 23 0
TOCKAZAREZ 23 ;
D_VIT_ZAGRADA 24 }
