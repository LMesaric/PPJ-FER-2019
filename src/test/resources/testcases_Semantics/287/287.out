<primarni_izraz> ::= IDN(20,s)
